### PR TITLE
[11.0][MIG] Add filter on mis builder preview - Analytic Account & Analytic Tags

### PR DESCRIPTION
**Pull request by apineux**
The purpose of this PR is to migrate the filter (Account analytic account) on the mis_builder preview. In addition we also migrate the PR: https://github.com/OCA/mis-builder/pull/228

### CODE REVIEW

File: mis_builder/models/mis_report_instance.py [hunk 1]
```diff
@@ -309,8 +309,12 @@ def _get_filter_domain_from_context(self):
...
             if values:
                 value = values.get('value')
-                operator = values.get('operator', '=')
-                filters.append((filter_name, operator, value))
+                if not value:
+                    continue
+                if not isinstance(value, list):
+                    value = [value]
+                for val in value:
+                    filters.append((filter_name, 'in', [val['id']]))
```
Comment (sbidoul) [inline mis_builder/models/mis_report_instance.py]: Same coment as on the 10.0 PR.

**Review by sbidoul — CHANGES_REQUESTED** [review]
The unit test for _get_filter_domain_from_context is on the `11.0-staging` branch. Could you rebase on that one?

**Comment by apineux** [comment]
This PR contains the similar features and corrections than https://github.com/OCA/mis-builder/pull/228

**Comment by sbidoul** [comment]
Thanks for this effort! I close this in favor of #245 which is simpler (using the `StandaloneFieldManagerMixin` to manage the field widgets, avoiding custom event handling), and preserves the same data structure for the `mis_report_filters` context key (important for compatibility with custom extensions).

**Pull request closed without merging**